providers/dns/hosttech/hosttech.go

```diff
@@ -114,9 +114,14 @@ func (d *DNSProvider) Present(domain, token, keyAuth string) error {
 		return fmt.Errorf("hosttech: could not find zone for domain %q (%s): %w", domain, authZone, err)
 	}
 
+	subDomain, err := dns01.ExtractSubDomain(fqdn, freemyip.RootDomain)
```

Comment:
Is `freemyip.RootDomain` really correct here?

Reply:
I was a copy-paste error